### ISSUE TITLE
Consistent cursor type for clickable UI components

### ISSUE THREAD

**Issue by vinaybadgujar102** [tijlleenders/ZinZen#1251]
**Describe the bug**
Currently, clickable UI components do not have a consistent cursor type. Some clickable elements show the pointer cursor upon hovering, while others do not. This issue is specifically for desktop users.

**To Reproduce**
1. Open the application.
2. Navigate to different pages or sections.
3. Hover over various clickable UI components.
4. Observe that some components show the pointer cursor, while others do not.

**Expected behavior**
All clickable UI components consistently display the pointer cursor upon hovering. This provides users with a clear indication that an element is interactive and clickable.

**Additional context**
Inconsistencies in the cursor type make it harder for users to identify clickable elements.

**Comment by Tushar-4781** [comment]
Yeah, i agree its just our primary focus was to get it done for mobile. As you can see the app is purely built for mobile. But we will soon start working on desktop version of this and we will resolve this issues you made. Nice catch
